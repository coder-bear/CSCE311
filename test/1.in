The quick brown fox sat on a log
The quirty brown fox sat on a rock
The quiet black fox sat on a dog
The quick tan hawk sat on a log
