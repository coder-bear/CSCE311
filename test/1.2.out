brown
The quick brown fox sat on a log
The quirty brown fox sat on a rock
